src/mono/browser/runtime/rollup.config.js

```diff
@@ -13,6 +13,12 @@ import fast_glob from "fast-glob";
 import gitCommitInfo from "git-commit-info";
 import MagicString from "magic-string";
 
+setTimeout(() => {
```

Comment:
what's the error message ? Or what's the problem ?
How/why does adding `.html` file influence rollup ?

Reply:
there is no error message, it just hangs forever. during the hang no code is running... it seems like the node instance just sits idle forever in a 'server' mode because rollup neglects to perform a process.exit when it's done.

Reply:
append `rollup -c --forceExit` in `src\mono\browser\runtime\package.json` will fix this